### PR TITLE
Added a placeholder chart in About Me page

### PR DESCRIPTION
**Pull request by casanted**
- Included a new drawChart() function to handle drawing a chart with hardcoded values about some animals. Will develop this to hold actual values of relevant data in the future

### CODE REVIEW

**Comment by casanted** [comment]
Screenshot 2020-07-09 at 12 06 58 PM

Screenshot of the placeholder chart

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -112,18 +112,53 @@ window.addEventListener('DOMContentLoaded', (event) => {
...
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Animal');
+  data.addColumn('number', 'Count');
+        data.addRows([
+          ['Lions', 10],
+          ['Tigers', 5],
+          ['Bears', 15]
+        ]);
+
+  const options = {
```
Comment (terencedignon) [inline portfolio/src/main/webapp/script.js:155]: Something we often do when adding placeholders is add a TODO. For example, `TODO(yourname): Update this with a data fetch`

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -24,20 +30,47 @@
...
+    PreparedQuery results = datastore.prepare(query);
+
+    int numComments = getLimit(request);
+
+    for (Entity entity : results.asIterable()) {
+      if (comments.size() == numComments) break;
+
+      String name = (String) entity.getProperty("name");
+      String text = (String) entity.getProperty("comment");
+      long timestamp = (long) entity.getProperty("timestamp");
```
Comment (terencedignon) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:68]: Hey, feel free to reach out to me directly if you have questions. Should this be in this PR? I was under the impression it was only about the placeholder chart.

Comment (casanted) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:68]: This is from an old PR, and I am not sure why it is included in this PR. The only file I changed was the script.js file to include the chart... I am not entirely sure what to do.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (casanted) [reply portfolio/src/main/webapp/script.js:155]: Gotcha!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (terencedignon) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Great, thank you for the fix.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -137,3 +137,29 @@ function createCommentElement(comment) {
 function deleteComments() {
   fetch('/delete-data', {method: 'POST'}).then(() => getComments());
 }
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Creates a chart and adds it to the page. */
+/* TODO (Carolyn): Update this chart to my actual bubble chart */
```
Comment (terencedignon) [inline portfolio/src/main/webapp/script.js:145]: Not a blocker, but an even better thing to do would be to open an issue in Github to track this change, then put the issue number here!

**Pull request merged**